samples/system-test/v3beta1/translate_batch_translate_text_beta.test.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright 2019, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const {assert} = require('chai');
+const {TranslationServiceClient} = require('@google-cloud/translate').v3beta1;
+const {Storage} = require('@google-cloud/storage');
+const execa = require('execa');
+const uuid = require('uuid');
+const exec = async cmd => (await execa.shell(cmd)).stdout;
+
+const REGION_TAG = 'translate_batch_translate_text_beta';
+
+describe(REGION_TAG, () => {
```

Comment:
Do not use `REGION_TAG` here, prefer writing out what the test suite intents to test.

Reply:
This is the latest test template, approved by @jmdobry 

See canonical test here:
https://github.com/googleapis/nodejs-dlp/blob/c5e165e2480aa8bb5af87769bdde3ab8dc4ed7cf/samples/system-test/dlp_inspect_image_file.test.js

( PR representing the latest canonical templates: https://github.com/googleapis/nodejs-dlp/pull/154 )

Reply:
Gotcha - sorry I wasn't aware 😅